### PR TITLE
Changes Morgue lights to use Red=Dead, and removes the now unneeded paper override

### PR DESCRIPTION
**Pull request by ghost**
Partially reverts but mostly reworks Skyrat-SS13/Skyrat-tg#16710 - used Revert to know I got every change needed and didnt mess up anywhere

I'd already seen this issue, and planned to simply change our colors to match - purple was gross anyways, we had green, and yellow is much more noticeable.
The mentioned PR was merged while I was working so I didn't have a chance to comment such on the PR, soooo just gonna toss this out as fast as I can. It got CLOSE, but changed the paper instead of solely fixing the lights. This'll just fix the lights.

image

The difference is negligible. BUT now we don't overwrite something we don't need to.

### CODE REVIEW

**Comment by ghost** [comment]
Wait
What?
Did I just totally misread this PR and I'm a 3am dumbass?

Oh no I see it still has the Purple, we can make that Red now. So it still technically applies just not as severely

File: modular_skyrat/modules/aesthetics/morgue/code/morgue.dm [hunk 1]
```diff
@@ -1,6 +1,2 @@
 /obj/structure/bodycontainer
 	icon = 'modular_skyrat/modules/aesthetics/morgue/icons/morgue.dmi'
```
Comment (ghost) [inline modular_skyrat/modules/aesthetics/morgue/code/morgue.dm:2]: There's a newline here according to the files I pushed, if somehow its not here I'm gonna cry

**Comment by Zergspower** [comment]
... This seems unnecessary?

**Comment by lessthnthree** [comment]
Purple for dead people is what SR has been used to and how the sprites were done. They were updated, and the documentation. #16710 was to update documentation and also because we had a sprite that alternated red/green, which is useless in the context of the documentation provided.

**Comment by ghost** [comment]
> ... This seems unnecessary?

> Purple for dead people is what SR has been used to and how the sprites were done. They were updated, and the documentation. https://github.com/Skyrat-SS13/Skyrat-tg/pull/16710 was to update documentation and also because we had a sprite that alternated red/green, which is useless in the context of the documentation provided.

I know the word 'revert' is scary in any context but the only 2 changes in this PR are changing the reddish purple to plain red, and removing the paper that's only change was "purple" in place of "red". I legitimately couldn't even see the change in the sprite from a distance when I did the purple to red.

I also already explained this on discord in the relevant dev channel, and you should respond to messages there instead of bringing comments onto the PR, because I was *inviting conversation*

**Comment by lessthnthree** [comment]
On discussion, yeah go for it.

**Comment by ghost** [comment]
My b, title could have been more helpful. I'll change it

**Review by Gandalf2k15 — APPROVED** [review]
ok

**Pull request merged**